gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode,
+  NxP,
+  NxTable,
+  NxTile,
+  NxH2,
+  NxH3,
+  NxTextLink,
+  NxStatefulAccordion,
+  NxAccordion,
+  NxWarningAlert
+} from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { GalleryTileFooter } from '../../gallery-components/GalleryTileFooter';
+
+import CodeExample from '../../CodeExample';
+
+const NxToastFullPageVariationsExampleCode = require('./NxToastFullPageVariationsExample?raw'),
+    NxToastFullPageSimpleExampleCode = require('./NxToastFullPageSimpleExample?raw');
+
+const NxToastPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Toasts are rendered on the top right corner of the screen, and will remain in place until the user
+        dismisses them individually. Each toast renders one <NxCode>NxAlert</NxCode>, which comes in four
+        variations: Error, Info, Warning, and Success. To find out more about <NxCode>NxAlert</NxCode>, you
+        can review the documentation <NxTextLink href="#/pages/Alert">here</NxTextLink>.
```

Comment:
It's considered bad for accessibility to make links out of just the word "here" or other similarly vague things.  IMO you could just remove this sentence and make a link out of `NxAlert` in the previous sentence.

Reply:
> It's considered bad for accessibility to make links out of just the word "here" or other similarly vague things. IMO you could just remove this sentence and make a link out of `NxAlert` in the previous sentence.

Thanks for letting me know re: accessibility. Updated here: [15eb707](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/15eb707d7f929f54fe60ea93471c01e7e3c716ea)